.github/workflows/example-components.yml

```diff
@@ -39,11 +39,9 @@ on:
   pull_request:
     branches: [main]
     paths:
-      - .github/workflows/examples-components.yml
+      - .github/workflows/example-components.yml
       - crates/wash/**
       - examples/rust/components/**
-      - examples/golang/components/**
-      - examples/tinygo/components/**
       - examples/python/components/**
```

Comment:
Removing these examples from the build at the same time, since they are covered by [wasmcloud/go](https://github.com/wasmcloud/go)